utilities_common/multi_asic.py

```diff
@@ -0,0 +1,159 @@
+import subprocess
+import json
+import click
+import functools
+from sonic_device_util import get_num_npus
+from sonic_device_util import get_namespaces
+from sonic_device_util import is_multi_npu
+from sonic_device_util import is_port_internal
+from sonic_device_util import is_port_channel_internal
+from sonic_device_util import is_bgp_session_internal
+from sonic_device_util import get_all_namespaces
+
+import swsssdk
+from swsssdk import ConfigDBConnector
+import argparse
+
+DEFAULT_NAMESPACE = ''
+DISPLAY_ALL = 'all'
+DISPLAY_EXTERNAL = 'frontend'
+PORT_CHANNEL_OBJ = 'PORT_CHANNEL'
+PORT_OBJ = 'PORT'
+BGP_NEIGH_OBJ = 'BGP_NEIGH'
+
+class MultiAsic(object):
+
+    def __init__(self, display_option=DISPLAY_ALL, namespace_option=None):
+        self.namespace_option = namespace_option
+        self.display_option = display_option
+        swsssdk.SonicDBConfig.load_sonic_global_db_config()
+        self.current_namespace = None
+
+    def connect_dbs_for_ns(self,namespace=DEFAULT_NAMESPACE):
+        '''
+        The function connects to the DBs for a given namespace and
+        returns the handle
+        If no namespace is provide, it will connect to the db in the
+        default namespace.
+        In case of multi ASIC, the default namespace in the database instance running the on the host
+        In case of single ASIC, the namespace has to DEFAULT_NAMESPACE
```

Comment:
"is the database instance"
"has to be .."

Reply:
Fixed in the latest commit